dask/dataframe/tests/test_dataframe.py

```diff
@@ -824,17 +824,20 @@ def test_assign():
                    d='string',
                    e=d.a.sum(),
                    f=d.a + d.b,
-                   g=lambda x: x.a + x.b)
+                   g=lambda x: x.a + x.b,
+                   dt=pd.datetime(2018, 2, 13))
```

Comment:
Sorry not seeing this earlier, could you change this to `pd.Timestamp`? `datetime.datetime` seems to be in pandas top-level namespace by accident...

Same with the other two cases below.

Otherwise, this looks good!

Reply:
ok will change this and your spacing comment below. Just for my own curiosity/understanding, can you elaborate on what the namespace issue is with using pd.datetime? You're saying `pd.datetime` is not really a pandas-wrapped datetime function, and is implicitly/accidentally just a call to `datetime.datetime`?

Reply:
Yep.

```
In [1]: import pandas

In [2]: import datetime

In [3]: pandas.datetime is datetime.datetime
Out[3]: True
```

Lots of packages accidentally expose their dependencies like this, but it's considered bad practice to rely on it. If / when pandas decides to clean up its API, this could be removed without warning.